$NetBSD: patch-aa,v 1.2 2011/10/14 02:05:01 jmmv Exp $

The structures/lexer directory rebuilds part of Boost.Wave by attempting to
access the raw Boost source files.  This is not appropriate for a package,
so just link against the installed Boost.Wave library.  We set LIBS_DIRS
from our package Makefile to avoid dealing with rpaths here.

--- src/Makefile.orig	2008-08-25 13:41:57.000000000 +0000
+++ src/Makefile
@@ -8,18 +8,15 @@ PLUGINS_OBJ = plugins/Interpreter.o plug
 	plugins/Transformations.o plugins/RootDirectory.o plugins/Parameters.o \
 	plugins/Reports.o plugins/cpptcl-1.1.4/cpptcl.o
 
-LIBS_DIRS = -L./structures/lexer -L${TCLLIB_DIR}
-LIBS = -l${TCL_LIB} -llexer
+LIBS_DIRS = undefined
+LIBS = -l${TCL_LIB} -lboost_wave -lboost_system
 
 ALL_OBJ = ${STRUCTURES_OBJ} ${PLUGINS_OBJ}
 
-vera++ : main.o structures_objects plugins_objects liblexer
+vera++ : main.o structures_objects plugins_objects
 	${CXX} -o $@ main.o ${ALL_OBJ} ${LIBS_DIRS} ${LIBS}
 
 
-liblexer :
-	make -C structures/lexer
-
 main.o : main.cpp
 	${CXX} -c -o $@ ${CXXFLAGS} $?
 
